### ISSUE TITLE
PHP Compatibility Checker Slider Revolution 6.2.22 bug

### ISSUE THREAD

**Issue by momoteb** [wpengine/phpcompat#324]
PHP Compatibility Checker
Scan blocked on Slider Revolution 6.2.22

I got this

Response status: 404

Message: The audit requested does not exis

**Comment by mindctrl** [comment]
@momoteb thanks for the report, and apologies for the issue. Looks to be a duplicate of https://github.com/wpengine/phpcompat/issues/323. I'll close this and we can track there instead.
